src/Functional.CQS.AOP.IoC.SimpleInjector/Models/ServiceAndImplementationType.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace Functional.CQS.AOP.IoC.SimpleInjector.Models
+{
+	/// <summary>
+	/// Encapsulates a service type and its associated implementation type.
+	/// </summary>
+	public struct ServiceAndImplementationType
+	{
+		/// <summary>
+		/// Initializes a new instance of the <see cref="ServiceAndImplementationType"/> struct.
+		/// </summary>
+		/// <param name="serviceType"></param>
```

Comment:
Documentation is missing for the `serviceType` and `implementationType` parameters.